content/client-server-api/modules/rich_replies.md

```diff
@@ -18,9 +19,22 @@ Additionally, a rich reply can reference any other event type as of v1.3.
 Previously, a rich reply could only reference another `m.room.message` event.
 {{% /boxes/note %}}
 
-When possible, events SHOULD include a [fallback representation](#fallbacks-for-rich-replies)
-to allow clients which do not render rich replies to still see something which
-appears to be a quoted reply.
+{{% boxes/note %}}
+{{% changed-in v="1.13" %}}
+In previous versions of the specification, rich replies could include a fallback
+format in the `body` and `formatted_body` for clients that do not support rich
+replies, this is no longer the case. Clients might still want to remove this
+fallback before rendering the event.
+
+To strip the fallback on the `body`, the client should iterate over each
+line of the string, removing any lines that start with the fallback
+prefix ("&gt; ", including the space, without quotes) and stopping when
+a line is encountered without the prefix. This prefix is known as the
+"fallback prefix sequence".
+
+To strip the fallback on the `formatted_body`, the client should remove
+the entirety of the `mx-reply` tag.
```

Comment:
Since the spec said that `mx-reply` was only allowed as the first tag in the formatted body, we should tell clients they only need to look there. Also, you can only do this if the format is org.matrix.custom.html.

```suggestion
To strip the fallback on the `formatted_body` of an `m.room.message` event with a `format` of
`org.matrix.custom.html`: if the`formatted_body` begins with an `<mx-reply>` start tag, the client 
should remove the entirety of the `<mx-reply>` element.
```

Reply:
Applied